index.js

```diff
@@ -0,0 +1,73 @@
+var dayTitle = document.querySelector(".t-day"),
+	dateTitle = document.querySelector(".t-date"),
+	monthTitle = document.querySelector(".month-year")
+var today = new Date();
+var days = [
+	'SUN',
+	'MON',
+	'TUE',
+	'WED',
+	'THU',
+	'FRI',
+	'SAT'
+]
+var months = [
+	'JAN',
+	'FEB',
+	'MAR',
+	'APR',
+	'MAY',
+	'JUN',
+	'JUL',
+	'AUG',
+	'SEP',
+	'OCT',
+	'NOV',
+	'DEC'
+]
+var currentDay = today.getDay();
+dayTitle.innerText = `${days[currentDay]}`;
```

Comment:
`days[currentDay]`는 항상 string을 반환하기 때문에
template literals를 사용할 필요가 없어 보입니다.
```suggestion
dayTitle.innerText = days[currentDay];
```